### ISSUE TITLE
Google Contacts missing column

### ISSUE THREAD

**Issue by joebordes** [tsolucio/corebos#256]
On these lines (at least):

https://github.com/tsolucio/corebos/blob/master/modules/Contacts/models/Module.php#L15
https://github.com/tsolucio/corebos/blob/master/modules/Contacts/models/Module.php#L32

there is a reference to a column that does not exist: its4you_googlesync4you_access.service

**Comment by joebordes** [comment]
> Regarding the table its4you_googlesync4you_access it actually has one column service and i didn't see any error on logs when removing sync or even when logging out the email account of google contact.

I'm sorry, you are correct, I forgot to activate the integration, but this got me thinking about the google calendar sync. If I understand this correctly we are saving both the OAuth for Calendar and Contacts in the its4you_googlesync4you_access table. In contacts it seems that the code searches explicitly for the correct line using the service column but the calendar code doesn't. Could it happen that one or the other recover the incorrect row and not use the correct values?

![googleintegrationservicecolumn](https://user-images.githubusercontent.com/1237191/38779291-6a8bf3b6-40c6-11e8-95eb-171ae710e26f.png)

**Comment by AlbanaCelepija** [comment]
Yes, it could happen for ex that Calendar gets the first row that could correspond to the configuration of Contacts. This is because the service field has been added after the implementation of the logic of Calendar and it has been using the logic of vtiger, so having both entities at the same table but getting the right configuration according to the type(Contacts/Calendar).So it remains either to add the value of the service at Calendar module or to separate the table where to save the parameters of Contacts. I think the first one could be a better solution.

**Comment by joebordes** [comment]
I agree. If you can add the service column to the calendar queries that would be wonderful.

I added Lorida in the conversation as she is the one who knows more about the Calendar sync, just so she is informed.

Thanks!
